README.md

```diff
@@ -233,6 +234,27 @@ https://github.com/openresty/lua-nginx-module/blob/master/.travis.yml
 
 [Back to TOC](#table-of-contents)
 
+reindex
+---------
+
+The `reindex` is used to unify the index format of the test files of each module under
```

Comment:
it's not only for `openresty` modules, but it can also be used anywhere.

Reply:
> it's not only for `openresty` modules, but it can also be used anywhere.

OK.  Bu it should use to `reindex` the test cases written by test::nginx,  or can it use in other way?

Reply:
done.